container/features/src/main/resources/features.xml

```diff
@@ -562,11 +562,20 @@
     </feature>
 
     <feature name="opennms-events-traps" description="OpenNMS :: Events :: Trap Daemon" version="${project.version}">
```

Comment:
This one is already exists in feature.xml, looks like it is an duplicate entry for TrapD which is defined below. Need to get rid of this.